synchronizer/config.py

```diff
@@ -44,7 +44,7 @@ class DevConfiguration(Configuration):
     DEV = True
     DEBUG = True
 
-    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:postgres@localhost/time'
+    SQLALCHEMY_DATABASE_URI = 'postgresql://postgres:postgres@localhost/time_sync_master'

```

Comment:
Restore previous comment.

Also, do not refactor files where you do not edit code (without an explicit request in the issue). This makes hard to review your changes.